<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockDialog</class>
 <widget class="QDialog" name="StockDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stock</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/qlot_512.svg</normaloff>../images/qlot_512.svg</iconset>
  </property>
  <widget class="QPushButton" name="valid_stock">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>-130</x>
     <y>30</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="text">
     <string>Nombre d'articles en stock :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_stock">
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>351</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="text">
     <string>Nombre d'articles livr√© le :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_article">
    <property name="maximumDate">
     <date>
      <year>7999</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>2014</year>
      <month>4</month>
      <day>27</day>
     </date>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_livre"/>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>valid_stock</sender>
   <signal>clicked()</signal>
   <receiver>StockDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
